sdk/python/feast/feature_view.py

```diff
@@ -182,6 +182,8 @@ def to_proto(self) -> FeatureViewProto:
             ttl_duration = Duration()
             ttl_duration.FromTimedelta(self.ttl)
 
+        print(f"Stream soruce: {self.stream_source}, {type(self.stream_source)}")
```

Comment:
Should this be removed?

Reply:
This has been removed in #1678

Reply:
Will remove and push, must have gotten added when I rebased.